eth/client.go

```diff
@@ -602,13 +573,7 @@ func (c *client) Vote(pollAddr ethcommon.Address, choiceID *big.Int) (*types.Tra
 		return nil, err
 	}
 
-	gl, gp := c.GetGasInfo()
-	opts, err := c.accountManager.CreateTransactOpts(gl, gp)
-	if err != nil {
-		return nil, err
-	}
-
-	return poll.Vote(opts, choiceID)
+	return poll.Vote(nil, choiceID)
```

Comment:
I think this might cause breakage. If the first argument is nil, then the underlying [Transact()](https://github.com/livepeer/go-livepeer/blob/04ef3afbc20797cf102fe55c925c385c74191dea/eth/contracts/poll.go#L226) call will receive a nil `*bind.TransactOpts`. Doesn't look like there are any nil checks in code for the underlying [Transact()](https://github.com/ethereum/go-ethereum/blob/19d7a37abb9f3b9bf1a94baf6bd8c7d5042e54f8/accounts/abi/bind/base.go#L207) call as it expects to receive a non-nil `*bind.TransactOpts`. So, if we pass nil here I think there would be a nil pointer error triggered.